src/core/agent/services/keriaNotificationService.ts

```diff
@@ -255,6 +255,8 @@ class KeriaNotificationService extends AgentService {
       shouldCreateRecord = await this.processMultiSigIcpNotification(notif);
     } else if (notif.a.r === NotificationRoute.MultiSigExn) {
       shouldCreateRecord = await this.processMultiSigExnNotification(notif);
+    } else if (notif.a.r === NotificationRoute.ExnIpexOffer) {
+      shouldCreateRecord = await this.processExnIpexOfferNotification(notif);
```

Comment:
hmm, since when are we processing `/ipex/offer` messages? It should just be apply, agree, grant. Or else multisig

Reply:
When executing the offer present ACDC, after the operation is done, an additional `ipex/offer` notification is being sent back. Since there is no action to handle these `ipex/offer` notifications, they still create a notification record on the user side (without any display interface). So, I added logic to handle these redundant notifications like the one mentioned above

Reply:
I see, might be similar to https://cardanofoundation.atlassian.net/browse/DTIS-1381 - I haven't fully investigated this yet. However, I'm not sure why we would want to create these notifications. Maybe this instead:
```
// @TODO - DTIS-1381; message appearing as notification after being multi-sig sent
shouldCreateRecord = false;
```

Once that spike is done I'll know how to handle it better. Either it's a bug, or we should skip handling it if we are the sender.